paypal.php

```diff
@@ -2041,6 +2044,7 @@ private function _postProcess()
             Configuration::updateValue(ConfigurationMap::COLOR, Tools::getValue(ConfigurationMap::COLOR));
             Configuration::updateValue(ConfigurationMap::ADVANCED_OPTIONS_INSTALLMENT, Tools::getValue(ConfigurationMap::ADVANCED_OPTIONS_INSTALLMENT));
             ConfigurationMap::setClientId(Tools::getValue(ConfigurationMap::CLIENT_ID, ''));
+            ConfigurationMap::setSecretId(Tools::getValue(ConfigurationMap::SECRET_ID, ''));
```

Comment:
Could you please find a validator before saving the value ?
By default apply Validate::isCleanHtml(). You could also verify a max length of the string.
It sould be great to apply the same valdiation on Client ID and other previous settings

Reply:
Done: Validation is now on clean HTML.
Added one validation too on colors to be sure we have real color sent.
The other ones are boolean and are not sent to the template as they are.